examples/tutorials/05_ordinary_torch_models.ipynb

```diff
@@ -0,0 +1,459 @@
+{
```

Comment:
<p>The very same four lines of code are used in many notebooks.</p><p>Write a simple plot_loss and use it everywhere</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>